### PR TITLE
Visa dependency bump

### PR DESCRIPTION
**Pull request by BR00KE**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -24,7 +24,7 @@ jobs:
           name: Validate release type
           command: /electrum/bin/validateReleaseType.sh
       - run:
-          name: Compile, Test, and Release
+          name: Compile, Test, and Release "-PpackagePluginRpm"
```
Comment (diedirk) [inline .circleci/config.yml:27]: why the `-PpackagePluginRpm`

File: release-notes.md [hunk 2]
```diff
@@ -1,40 +1,49 @@
 # Message Masking SDK Release Notes
 
+## Version 2.2.0 - 2 December 2021
+
+### Fixed
+
+* Bumped `json-path.version` from v2.3.0 to v2.6.0. This was updated so that the `json-smart` compile dependency is
+  updated to v2.4.7 which has no known vulnerabilities.
```
Comment (diedirk) [inline release-notes.md:8]: This reads a bit strange for someone that doesn't have context. Perhaps state that earlier versions had known vulnerabilities and this one doesn't.

**Review by diedirk — APPROVED** [review]
Two comments but only around the docs

File: .circleci/config.yml (hunk 1 above)
Comment (BR00KE) [reply .circleci/config.yml:27]: not actually super sure - but it's from the example circle file on this wiki https://electrum.atlassian.net/wiki/spaces/EE/pages/585793610/How+to+set+up+and+configure+my+project+for+CircleCI
can remove?

**Pull request merged**